### PR TITLE
Create a BigintRange when values list are all duplicates

### PR DESCRIPTION
**Pull request by ericyuliu**
TEST_F(InPredicateTest, bigIntDuplicate)
    Before: 
    Exception thrown for from BigintValuesUsingBitmask when min = max
    After:
    BigintRange is used, and Unit Test passed

### CODE REVIEW

File: velox/type/Filter.cpp [hunk 1]
```diff
@@ -1054,6 +1054,9 @@ std::unique_ptr<Filter> createBigintValuesFilter(
         return std::make_unique<NegatedBigintValuesUsingBitmask>(
             min, max, values, nullAllowed);
       }
+      if( min == max) {
```
Comment (Yuhta) [inline velox/type/Filter.cpp:1057]: nit: `if (min == max)`

Comment (Yuhta) [inline velox/type/Filter.cpp:1057]: Also we should check it before line 1022, no?

Comment (ericyuliu) [reply velox/type/Filter.cpp:1057]: Hi Jimmy,
good catch, it is more efficient to check min == max earlier.
I moved it to line# 1039, to use the min max;

File: velox/type/Filter.cpp [hunk 2]
```diff
@@ -1036,6 +1036,12 @@ std::unique_ptr<Filter> createBigintValuesFilter(
       min = values[i];
     }
   }
+  if (min == max) {
```
Comment (mbasmanova) [inline velox/type/Filter.cpp:1039]: Per offline discussion, this change should not be needed.

**Comment by ericyuliu** [comment]
close since it is fixed in D47784943

**Pull request closed without merging**